### PR TITLE
task/KAH-955 (Fix CircleCI integration tests config for kaholo-enterprise) - revert Typescript

### PR DESCRIPTION
**Pull request by kbuchacz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
   "name": "kaholo-agent",
   "version": "1.3.5",
   "description": "Agent for Kaholo",
-  "main": "app.ts",
+  "main": "index.js",
```
Comment (arispen) [inline package.json:5]: change to app.js

File: app.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const enviormentConfig = require('./config/environment');
+const expressConfig = require('./config/express');
+
+const bootstrap = require("./utils/bootstrap");
+
+enviormentConfig().then(()=>{
+    return expressConfig();
+}).then((serevr)=>{
```
Comment (arispen) [inline app.js:8]: you can fix typos here or I can in my next pull request

File: service-mac.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const serviceInstaller = require('./utils/service-installer');
+
+var Service = require("node-mac").Service;
+
+serviceInstaller(Service);
```
Comment (arispen) [inline service-mac.js:5]: I've heard we can delete these?

File: service-windows.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const serviceInstaller = require('./utils/service-installer');
```
Comment (arispen) [inline service-windows.js:1]: I've heard we can delete these?

**Review by arispen — APPROVED** [review]
minor comments to code that was written by other people ;)

File: service-mac.js (hunk 3 above)
Comment (kbuchacz) [reply service-mac.js:5]: Yeah, true. That's what @theflexbird said according to the installation process.

File: app.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const enviormentConfig = require('./config/environment');
+const expressConfig = require('./config/express');
+
+const bootstrap = require("./utils/bootstrap");
+
+enviormentConfig().then(()=>{
+    return expressConfig();
```
Comment (ChBernat) [inline app.js:7]: Not formatted, you could also shorten it.

File: app.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+}).then(()=>{
```
Comment (ChBernat) [inline app.js:8]: Not formatted, you could also shorten it.

**Review by ChBernat — CHANGES_REQUESTED** [review]

**Review by ChBernat — APPROVED** [review]

**Pull request merged**